<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memorize</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="WordList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd_Word"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actiontutorial"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionTheUserWW"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionSChinese"/>
    <addaction name="actionTChinese"/>
    <addaction name="actionTibetan"/>
    <addaction name="actionUyghur"/>
    <addaction name="actionKazakh"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionFrench"/>
    <addaction name="actionSpanish"/>
    <addaction name="actionRussian"/>
    <addaction name="actionArabic"/>
    <addaction name="actionSyriac"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Word"/>
   <addaction name="actionHome"/>
   <addaction name="actionTest"/>
   <addaction name="actionStatistics"/>
   <addaction name="actionDelete"/>
   <addaction name="actionFlashcard"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_Word">
   <property name="text">
    <string>&amp;Add Word</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>&amp;Test</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>&amp;Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>&amp;Statistics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionFlashcard">
   <property name="text">
    <string>&amp;Flashcard</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionTheUserWW">
   <property name="text">
    <string>TheUserWW</string>
   </property>
  </action>
  <action name="actiontutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionSChinese">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionmale_voice">
   <property name="text">
    <string>male voice</string>
   </property>
  </action>
  <action name="actionfemale_voice">
   <property name="text">
    <string>female voice</string>
   </property>
  </action>
  <action name="actionEnglish_2">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionEnglish_3">
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionEnglish_4">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionTChinese">
   <property name="text">
    <string>繁體中文</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="text">
    <string>العربية</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionTibetan">
   <property name="text">
    <string>བོད་སྐད་</string>
   </property>
  </action>
  <action name="actionUyghur">
   <property name="text">
    <string>ئۇيغۇرچە</string>
   </property>
  </action>
  <action name="actionKazakh">
   <property name="text">
    <string>Қазақша</string>
   </property>
  </action>
  <action name="actionSyriac">
   <property name="text">
    <string>ܣܘܪܝܝܐ</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Español</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
